app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,140 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.domain.Task;
+import com.codesoom.assignment.dto.TaskViewDto;
+import com.codesoom.assignment.dto.TaskEditDto;
+import com.codesoom.assignment.dto.TaskSaveDto;
+import com.codesoom.assignment.service.TaskService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ *  할 일에 대한 클라이언트 요청 지점입니다.
+ */
```

Comment:
컨트롤러의 역할 중 하나는 HTTP 프로토콜이 도메인 레이어까지 침투되지 않도록 막아주는 것이기도 합니다.
```suggestion
/**
 *  할 일에 대한 HTTP 요청을 처리합니다.
 */
```